### PR TITLE
Int 2918 part 2

### PR DESCRIPTION
**Pull request by hlmuludiang**

### CODE REVIEW

File: postman/Get Broadcast Stats.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+				{
+					"listen": "test",
+					"script": {
+						"id": "e238f70c-3010-41a2-ba60-6fd4e312850d",
+						"exec": [
+							"var jsonData = pm.response.json();",
+							"pm.globals.set(\"broadcast_self_link\", '');",
+							"",
+							"pm.test(\"Successfully retreived sent broadcasts from list\", function() {",
+							"    if (pm.expect(jsonData.entries.length).to.be.above(0)) {",
```
Comment (cknave) [inline postman/Get Broadcast Stats.postman_collection.json]: You don't need to wrap the expect in an if statement. It will stop the test if it fails.

Comment (hlmuludiang) [reply postman/Get Broadcast Stats.postman_collection.json]: Updated

File: postman/Get Clicks.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,572 @@
...
+						{
+							"listen": "test",
+							"script": {
+								"id": "d2b1b30c-3aec-48cc-9f35-c1ab0683ee45",
+								"exec": [
+									"var jsonData = pm.response.json();",
+									"",
+									"pm.test(\"Successfully got a followup email\", function() {",
+									"    var followup_entry = Math.floor((Math.random() * jsonData.entries.length ));",
+									"    pm.globals.set(\"followupId\", jsonData.entries[pm.globals.get(\"followup_entry\")].id);",
```
Comment (cknave) [inline postman/Get Clicks.postman_collection.json]: not a global

```suggestion
									"    pm.globals.set("followupId", jsonData.entries[followup_entry].id);",
```

File: postman/Get Clicks.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,572 @@
...
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "4d22c8fa-8695-4098-a275-10658e8aaea5",
+								"exec": [
+									"var jsonData = pm.response.json();",
+									"",
+									"pm.test(\"Successfully got a  random list\", function() {",
+									"    if (pm.expect(jsonData.entries.length).to.be.above(0)) {",
```
Comment (cknave) [inline postman/Get Clicks.postman_collection.json]: You don't need to wrap the expect in an if statement. It will stop the test if it fails.

Comment (hlmuludiang) [reply postman/Get Clicks.postman_collection.json]: Removed pm.expect...

File: postman/Get Clicks.postman_collection.json (hunk 2 above)
Comment (hlmuludiang) [reply postman/Get Clicks.postman_collection.json]: Fixed

File: postman/Get Clicks.postman_collection.json (hunk 3 above)
Comment (cknave) [reply postman/Get Clicks.postman_collection.json]: It's better with the expect! What I mean is that you don't have to check the result of the expect yourself:

```
pm.expect(jsonData.entries.length).to.be.above(0);
var list_entry = Math.floor((Math.random() * jsonData.entries.length));
pm.globals.set("listId", jsonData.entries[list_entry].id );
```

**Pull request merged**